### DISCUSSION TITLE
最近好冷

### DISCUSSION THREAD

**Discussion by Taoyi-Chiang** [1101-GS4719-C/Python-Community-GS4719-C#3654] [閒聊八卦]
我一直很好奇，大家都說洋蔥式穿搭，可是有些發熱衣什麼的真的很難洋蔥式穿搭。
然後常常在室內穿著外套又太熱，不穿外套又會冷。
冬天的衣服又都很黑，看到暗暗的顏色感覺心情就更不好了。
最後抱怨一下到底為什麼中央可以這麼冷，到底什麼荒謬的苦寒之地。

**Comment by Elaine4488** [comment]
室內外溫差我也覺得很困擾，騎車的話也還要再多準備防風外套跟手套，不然真的會超級冷

**Comment by Mminniehuang** [comment]
面對超冷的冬天我一直是洋蔥式穿法，薄長袖+帽T+刷毛外套+厚外套，褲子襪子穿兩層。
我沒有在穿發熱衣不太清楚，不然我覺得洋蔥法超棒，在任何環境都可以視情況調整，不過是沒有太覺得說在室內多穿一件太熱的問題，可能我真的太怕冷了...
最近真的真的好冷，尤其是鼻子吸到冷冷的空氣超級不舒服，想一直待在被窩裡面，但知道自己會因為太舒服怠惰，期末考就完蛋了所以不得不離開......
